index.js

```diff
@@ -99,14 +99,15 @@ MenuDroppoComponent.prototype.manageListeners = function() {
 
 MenuDroppoComponent.prototype.componentDidMount = function() {
   if (this) {
-    window.addEventListener('click', this.windowWasClicked.bind(this))
+    this.windowClickHandler = this.windowWasClicked.bind(this);
+    window.addEventListener('click', this.windowClickHandler)
     var container = findDOMNode(this)
     this.container = container
   }
 }
 
 MenuDroppoComponent.prototype.componentWillUnmount = function() {
-  window.removeEventListener('click', this.windowWasClicked.bind(this))
```

Comment:
As-written, this line will never remove a listener from window, because calling `Function.prototype.bind` twice will always return a new function.

For an example, try this in console:
```
const x = () => {};
x.bind({}) === x.bind({}) 
// false
```